.travis.yml

```diff
@@ -18,9 +18,13 @@ matrix:
     - {python: 3.6, env: TOXENV=py36-poll}
     - {python: 3.6, env: TOXENV=py36-selects}
 
-    - {python: 3.7, sudo: required, dist: xenial, env: TOXENV=py37-epolls}
-    - {python: 3.7, sudo: required, dist: xenial, env: TOXENV=py37-poll}
-    - {python: 3.7, sudo: required, dist: xenial, env: TOXENV=py37-selects}
+    - {python: 3.7, sudo: required, env: TOXENV=py37-epolls}
```

Comment:
not related to 3.8

Reply:
It is routine tidy up.  I will split it into a separate commit in this PR.

Reply:
Tidying up like this belongs in a separate PR as far as I'm concerned, pollutes the history otherwise.

Reply:
I was angry at first, but few deep breaths later, it even makes sense. In the end it would be a 3.8 commit and tidy commit, like after proper merge (which is of course fast forward without merge commit). So github PR history is polluted, but git history is fine.

@jstasiak let's agree to make it quicker this one time.
@jayvdb please make unrelated changes in separate patches in future, no PR-on-PR thing. It may seem stupid rules, but keeping patches with small scope really helps to make fast and correct decisions. It's related to limited time we can invest into volunteer maintaining, I suppose you understand.

Reply:
Sure, fair enough.